src/components/lib/cos_kernel_api.c

```diff
@@ -17,11 +17,20 @@
 #define printd(...)
 #endif
 
+/* RSK -- move this to proper location, probably consts.h
+ * At this point I'm following runyu's lead but I don't think is
+ * a good long term solution
+ * */
+#define SUPER_PAGE_ORDER 22
```

Comment:
This should be in `consts.h`, and (as I said above), we need a way to specify the different page sizes in an architecture-specific way.